package.json

```diff
@@ -2,19 +2,18 @@
   "name": "@govtechsg/open-attestation",
   "version": "0.0.0-development",
   "description": "## Usage",
-  "type": "module",
-  "exports": {
-    ".": "./dist/index.js"
-  },
-  "types": "dist/index.d.ts",
-  "engines": {
-    "node": ">=12"
-  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "browser": "dist/index.umd.js",
+  "types": "dist/types/index.d.ts",
   "scripts": {
     "benchmark:qr-code": "ts-node --transpile-only benchmarks/qr-code",
-    "build": "npm run build:clean && npm run build:esm",
-    "build:esm": "tsc --outDir dist --declaration --project ./tsconfig.prod.json",
-    "build:clean": "rm -rf dist/",
+    "build": "npm run clean && npm run build:cjs && npm run build:esm && npm run build:umd && npm run build:type",
+    "build:cjs": "tsc --module commonjs --outDir dist/cjs --project ./tsconfig.prod.json",
+    "build:esm": "tsc --module es2015 --outDir dist/esm --project ./tsconfig.prod.json",
+    "build:type": "tsc -d --emitDeclarationOnly --outDir dist/types --project ./tsconfig.prod.json",
+    "build:umd": "rollup -c",
```

Comment:
since rollup is building from `./dist/esm/index.js`, should we do
```
"build:umd": "npm run build:cjs && rollup -c",
```

Reply:
need to update the circle i to install rollup

Reply:
hum no I will keep it as it was before

Reply:
for rollup I forgot to put it back. Fixed